### ISSUE TITLE
KP Dossier Visuals - Status Tracker

### ISSUE THREAD

**Issue by vanamala-narasimhan** [ICPI/HSS-KP#28]
https://github.com/pepfar-datim/Analytics/issues/2483 - Analytics Repo overall tracker

The Key Populations Q4 "Dossier" is a collection of visuals across multiple Q3/Q4 dossiers listed below, as well as their analytics repo issue link:

- [x] CC Single OU - Changes done and Janice to resave over the original dossier. https://github.com/pepfar-datim/Analytics/issues/2448
- [x] CC All OU - Changes done on the dossier in the Q4 folder. https://github.com/pepfar-datim/Analytics/issues/2448
- [ ] CC Site - (pending on approval) https://github.com/pepfar-datim/Analytics/issues/2448
- [x] Prevention - on hold. https://github.com/pepfar-datim/Analytics/issues/2420
- [x] Testing Single OU - Changes done in a copy under Key population folder
- [x] Testing All OU - Changes done in a copy under Key population folder
- [x] Treatment Single OU - Changes done in the dossier under Treatment folder https://github.com/pepfar-datim/Analytics/issues/2193

Requirements document can be found linked to Analytics Repo tracker

**Comment by vanamala-narasimhan** [comment]
@janholcombe all KP related dossiers ready to go w/ CC SIte deferred to Q1

**Comment by vanamala-narasimhan** [comment]
All Q4 KP dossiers ready w exception of CC Site deferred to Q1
